packages/ember-metal/lib/property_get.js

```diff
@@ -74,24 +68,20 @@ export function _getPath(root, path) {
   let parts = path.split('.');
 
   for (let i = 0; i < parts.length; i++) {
-    if (!isGettable(obj)) {
+    if (obj === undefined || obj === null || obj.isDestroyed) {
```

Comment:
I don't think we want to hard code `.isDestroyed` check here. In theory it would be better to `let m = peekMeta(obj); m.isDestroyed()`.

Reply:
it is how done before https://github.com/emberjs/ember.js/blob/master/packages/ember-metal/lib/property_get.js#L83 <- just moved this check to top
I think checking `isDestroyed` with `meta` worth another PR since hardcoded `isDestroyed` check is done in more places

Reply:
I will do follow up PR to use `meta.isDestroyed()` does it sound good ? or should I do it with this PR

Reply:
Ya, sounds good to do separately